scaffolds/hasura/scaffold.tsx

```diff
@@ -11,10 +12,16 @@ type HasuraData = NpmClientPrompt.Data &
     jwtSecret: string;
   };
 
+const generateJwtSecret = () => {
```

Comment:
Stylistic nit: since this is at the top-level of the module, let's use `function generateJwtSecret() { ... }` instead of `const generateJwtSecret = () => { ... }`